### PR TITLE
Add test 2847

### PR DESCRIPTION
**Pull request by maryvictol**
This PR adds testcase and e2e-test for Introduce extended attributes support in pipe fuse #2847

### CODE REVIEW

File: docs/testcases/UI/Pipe_CLI/2847_introduce extended attributes support.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+**Preparations**:
+1. Create object storage `storage1`
+2. In the `storage1` create `file1`
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the admin user from the prerequisites | |
+| 2 | Open the Tools page | |
+| 3 | Select `centos` tool | |
+| 4 | Run tool with *Custom settings* | |
```
Comment (NShaforostov) [inline docs/testcases/UI/Pipe_CLI/2847_introduce extended attributes support.md]: I suggest to change the step to 'Launch the tool on on-demand instance'.

File: docs/testcases/UI/Pipe_CLI/2847_introduce extended attributes support.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+| 5 | At the Runs page, click the just-launched run | |
+| 6 | Wait until the SSH hyperlink appears | |
+| 7 | Click the SSH hyperlink | |
+| 8 | In the opened tab, enter and perform the command `yum install attr` | |
+| 9 | In the opened tab, enter and perform the command `setfattr -n user.key1 -v value1 cloud-data/storage1/file1` | |
+| 10 | In the opened tab, enter and perform the command `setfattr -n user.key2 -v value2 cloud-data/storage1/file1` | |
+| 11 | Enter and perform the command `getfattr cloud-data/storage1/file1` | The output contains <br> `file: cloud-data/storage1/file1` <br> `user.key` <br> `user.key2` |
```
Comment (NShaforostov) [inline docs/testcases/UI/Pipe_CLI/2847_introduce extended attributes support.md]: > `file: cloud-data/storage1/file1` <br> `user.key`

Fix to:
`file: cloud-data/storage1/file1` <br> `user1.key`

**Review by NShaforostov — APPROVED** [review]
Fix the minor notes before the merge, please (see the comments).

**Review by kamyshova — APPROVED** [review]

File: docs/testcases/UI/Pipe_CLI/2847_introduce extended attributes support.md (hunk 1 above)
Comment (maryvictol) [reply docs/testcases/UI/Pipe_CLI/2847_introduce extended attributes support.md]: Done

File: docs/testcases/UI/Pipe_CLI/2847_introduce extended attributes support.md (hunk 2 above)
Comment (maryvictol) [reply docs/testcases/UI/Pipe_CLI/2847_introduce extended attributes support.md]: Done

**Pull request merged**